packages/clients/textLocator/src/locales.ts

```diff
@@ -0,0 +1,121 @@
+        },
+        addressSearch: {
+          resultInfo: {
+            name: 'Name',
+            type: 'Typ',
+            language: 'Sprache',
+            timeFrame: 'Zeitraum',
+          },
+        },
+        attributions: {
+          [openStreetMap]: `$t(textLocator.layers.${openStreetMap}): © <a href='https://www.openstreetmap.org/copyright' target='_blank'>OpenStreetMap</a> contributors`,
+          [openSeaMap]: `$t(textLocator.layers.${openSeaMap}): © <a href='https://www.openseamap.org/ target='_blank''>OpenSeaMap</a>`,
+          [mdiSeaNames]: `$t(textLocator.layers.${mdiSeaNames}): © <a href='https://projekt.mdi-de.org/kuesten-gazetteer/ target='_blank''>MDI DE</a>`,
+          [wmtsTopplusOpenWeb]: `$t(textLocator.layers.${wmtsTopplusOpenWeb}): © Bundesamt für Kartographie und Geodäsie {{YEAR}}`,
+          [wmtsTopplusOpenWebGrey]: `$t(textLocator.layers.${wmtsTopplusOpenWebGrey}): © Bundesamt für Kartographie und Geodäsie {{YEAR}}`,
+          [wmtsTopplusOpenLight]: `$t(textLocator.layers.${wmtsTopplusOpenLight}): © Bundesamt für Kartographie und Geodäsie {{YEAR}}`,
+          [wmtsTopplusOpenLightGrey]: `$t(textLocator.layers.${wmtsTopplusOpenLightGrey}): © Bundesamt für Kartographie und Geodäsie {{YEAR}}`,
+          [aerial]:
+            '© Europäische Union, enthält veränderte Copernicus Sentinel-Daten ({{YEAR}})',
+          kuestengazetteer:
+            '<br>Zur Ortssuche wird der <a href="https://projekt.mdi-de.org/kuesten-gazetteer/" target="_blank">Küstengazetteer</a> eingesetzt.',
+          static:
+            '<a href="https://github.com/Dataport/polar/blob/main/LEGALNOTICE.md" target="_blank">Impressum</a>',
+        },
+        info: {
+          noLiteratureFound: 'Es wurden keine Texte zu diesen Orten gefunden.',
+          noGeometriesFound:
+            'Es wurden keine Orte zu dieser Geometrie gefunden.',
+          loadingTime:
+            'Es gibt viele Ergebnisse zu der letzten Anfrage. Der Ladevorgang kann einen Moment länger dauern.',
+        },
+        error: {
+          searchCoastalGazetteer:
+            'Die Suche ist mit einem unbekannten Fehler fehlgeschlagen. Bitte versuchen Sie es später erneut.',
+        },
+      },
+      plugins: {
+        addressSearch: {
+          defaultGroup: 'Ortssuche',
+        },
+      },
+    },
+  },
+  // NOTE: English translation not yet required and may be incomplete
+  {
+    type: 'en',
+    resources: {
+      textLocator: {
+        layers: {
+          [openStreetMap]: 'OpenStreetMap',
+          [openSeaMap]: 'OpenSeaMap',
+          [mdiSeaNames]: 'Coastal name service',
+          [wmtsTopplusOpenWeb]: 'TopPlusOpen (Web)',
+          [wmtsTopplusOpenWebGrey]: 'TopPlusOpen (Web, Grey)',
+          [wmtsTopplusOpenLight]: 'TopPlusOpen (Light)',
+          [wmtsTopplusOpenLightGrey]: 'TopPlusOpen (Light, Grey)',
+          [aerial]: 'Luftbilder Sen2Europe',
+        },
+        addressSearch: {
+          unnamed: 'Unnamed',
+        },
```

Comment:
Yep, I stopped updating it after it became clear no translation was desired, as denoted in `// NOTE: English translation not yet required and may be incomplete`. I figured to keep the translated parts in case it's to be done in a later iteration.